x11-misc/revelation/revelation-0.5.5.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+
+inherit gnome2-utils python-single-r1 meson xdg
+
+DESCRIPTION="A password manager for GNOME"
+HOMEPAGE="https://revelation.olasagasti.info/ https://github.com/mikelolasagasti/revelation"
+SRC_URI="https://github.com/mikelolasagasti/revelation/releases/download/${P}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# Upstream does not provide any test suite.
+RESTRICT="test"
+
+RDEPEND="${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/pycryptodome[${PYTHON_USEDEP}]
+		dev-python/pygobject[${PYTHON_USEDEP}]
+		dev-libs/libpwquality[python,${PYTHON_USEDEP}]
+		dev-python/defusedxml[${PYTHON_USEDEP}]
+	')
+	x11-libs/gtk+:3
+	dev-libs/glib
+	dev-libs/gobject-introspection
+"
+
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	find -name '*.py' -exec \
```

Comment:
It'd be nice to patch this upstream, e.g. https://github.com/p0dalirius/pydsinternals/commit/05904ec91b230150c04ba10bb4556450eead5438.

Reply:
I looked into this, and honestly am not sure I want to do this right now. If I were in upstream's shoes, I wouldn't want to commit to supporting both libraries unless I ran CI for it... which makes me not want to ask someone else to accept a patch to do just that :).

Reply:
Well, the point is it's trivial, and it's a simple fallback. But alright.

Reply:
I'm just a little peculiar about making changes that impact crypto things unless I'm :100: sure that it's not doing anything problematic (like reducing the test coverage). If I don't have the time to do the work to ensure upstream tests with both libraries, I don't want to be responsible for introducing that change upstream.